src/tactic/ui/tools/trigger_wdg.py

```diff
@@ -323,7 +323,16 @@ def get_trigger_wdg(my, trigger, index=1):
         } )
         return trigger_div
 
-
+    def get_trigger_data(my, trigger):
+        '''returns a the python dictionary representing
+           data attribute of trigger sObject.'''
+        raw_data = trigger.get_value("data", no_exception=True)
+        try:
+            data = jsonloads(raw_data)
+        except:
+            data = {}
+        return data
+    get_trigger_data = classmethod(get_trigger_data)
```

Comment:
for classmethod the convention is to use "cls" instead of "my".. but here , I recommend using staticmethod instead and you only need to pass in (trigger) in the argument list